jenkins/defaults.yaml

```diff
@@ -12,9 +12,9 @@ jenkins:
   home: /var/lib/jenkins
   java_args: -Djava.awt.headless=true
   java_executable: /usr/bin/java
-  jenkins_args:
+  jenkins_args: ''
   max_open_files: 32768
-  umask: 027
+  umask: '027'
```

Comment:
yes, salt interpret 027 value as int not as string '027', so there is printed e.g. 23 (I'm not sure now with exact value, but it's not string 027)